src/main/java/teammates/common/datatransfer/questions/FeedbackMsqQuestionDetails.java

```diff
@@ -457,21 +473,31 @@ public String getQuestionResultStatisticsHtml(List<FeedbackResponseAttributes> r
         }
 
         if (!isContainsNonEmptyResponse) {
-            return "";
+            return -1;
         }
 
-        DecimalFormat df = new DecimalFormat("#.##");
+        return numChoicesSelected;
+    }
 
-        StringBuilder fragments = new StringBuilder();
-        for (Entry<String, Integer> entry : answerFrequency.entrySet()) {
-            fragments.append(Templates.populateTemplate(FormTemplates.MCQ_RESULT_STATS_OPTIONFRAGMENT,
-                                Slots.MCQ_CHOICE_VALUE, entry.getKey(),
-                                Slots.COUNT, entry.getValue().toString(),
-                                Slots.PERCENTAGE, df.format(100 * (double) entry.getValue() / numChoicesSelected)));
+    private int checkContainsNonEmptyResponse(List<String> answerStrings, Map<String, Integer> answerFrequency) {
```

Comment:
Similarly here. One would not expect a method named `check*` to return an int, no?

Also, if the method is only used once, you might not need to abstract it out. Do not abstract out a method or routine just for the sake of it (e.g. to make the call site look shorter). You need to be able to explain the abstracted out method on its own.